core/ledger/pvtdatastorage/store_impl_test.go

```diff
@@ -829,24 +825,6 @@ func TestRollBack(t *testing.T) {
 	assert.True(testDataKeyExists(t, store, datakeyTx0))
 	assert.True(testDataKeyExists(t, store, datakeyTx5))
 	assert.True(testMissingDataKeyExists(t, store, eligibleMissingdatakey))
```

Comment:
Since PR removed Rollback() from store interface, is there any value to keep TestRollBack?